inbm/cloudadapter-agent/Makefile

```diff
@@ -2,6 +2,14 @@
 # SPDX-License-Identifier: Apache-2.0
 
 .PHONY: tests # `make tests` doesn't run because of the tests directory
+
+.PHONY: documentation # prevent dup issues from calling documentation folder in higher level dir 
+
+documentation:
+	mkdir -p documentation
+	pydoc -w `find . -iname '*.py'`
```

Comment:
I tried this and it didn't work. Error: 

```
/bin/sh: pydoc: command not found
```
Could you add instructions on how to install pydoc?